pkg/dbutil/common.go

```diff
@@ -606,6 +604,60 @@ func GetDBVersion(ctx context.Context, db *sql.DB) (string, error) {
 	return "", ErrVersionNotFound
 }
 
+// GetGlobalVariable gets server's global variable
+func GetGlobalVariable(db *sql.DB, variable string) (value string, err error) {
+	query := fmt.Sprintf("SHOW GLOBAL VARIABLES LIKE '%s'", variable)
+	rows, err := db.Query(query)
+
+	if err != nil {
+		return "", errors.Trace(err)
+	}
+	defer rows.Close()
+
+	// Show an example.
+	/*
+		mysql> SHOW GLOBAL VARIABLES LIKE "binlog_format";
+		+---------------+-------+
+		| Variable_name | Value |
+		+---------------+-------+
+		| binlog_format | ROW   |
+		+---------------+-------+
+	*/
+
+	for rows.Next() {
+		err = rows.Scan(&variable, &value)
+		if err != nil {
+			return "", errors.Trace(err)
+		}
+	}
+
+	if rows.Err() != nil {
+		return "", errors.Trace(err)
+	}
+
+	return value, nil
+}
+
+// GetSQLMode returns sql_mode.
+func GetSQLMode(db *sql.DB) (tmysql.SQLMode, error) {
+	sqlMode, err := GetGlobalVariable(db, "sql_mode")
```

Comment:
Should be session variable because now we support session config for db？

Reply:
done in https://github.com/pingcap/tidb-tools/pull/381/commits/3dccc006997826ff0344dc233afda8776d223410